src/content-distribution/distribute/style.scss

```diff
@@ -0,0 +1,55 @@
+@use "~@wordpress/base-styles/colors";
+
+.newspack-network-distribute {
+	height: calc(100% - 47px);
+	.components-panel {
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+	}
+	.components-panel__body {
+		border: 0;
+		&.distribute-header {
+			p {
+				color: colors.$gray-700;
+			}
+			.components-base-control__field {
+				margin: 0;
+			}
+			input[type="text"] {
+				padding: 0 16px;
+				height: 40px;
+			}
+		}
+		&.distribute-body {
+			flex: 1 1 100%;
+			overflow-y: auto;
+			padding-top: 0;
+			.components-checkbox-control {
+				--checkbox-input-size: 20px;
+				height: 44px;
+				display: flex;
+				align-items: center;
+				.components-base-control__field {
+					margin-bottom: 0;
+				}
+				.components-checkbox-control__input-container {
+					margin-right: 1em;
```

Comment:
This can be removed. Core's right margin is fine.

Reply:
Updated in a1a94cafc0997adef87e181488ef094a4eb13bc1